lib/npg_qc/autoqc/checks/pulldown_metrics.pm

```diff
@@ -142,33 +172,40 @@ override 'execute' => sub {
     return 1;
 };
 
+has 'output_file' => (
```

Comment:
POD is absent for this attribute. The code is better moved up to be next to other public attributes.